int32 roomID 
int32 battryLevel
string roomName
